packages/js/product-editor/src/hooks/use-feedback-bar/use-feedback-bar.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { resolveSelect, useDispatch } from '@wordpress/data';
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import { PRODUCT_EDITOR_SHOW_FEEDBACK_BAR_OPTION_NAME } from '../../constants';
+
+async function wasFeedbackBarPreviouslyHidden(): Promise< boolean > {
+	const optionValue: string = await resolveSelect(
+		OPTIONS_STORE_NAME
+	).getOption( PRODUCT_EDITOR_SHOW_FEEDBACK_BAR_OPTION_NAME );
+	return optionValue === 'no';
+}
+
+export const useFeedbackBar = () => {
+	const { updateOptions } = useDispatch( OPTIONS_STORE_NAME );
+
+	const showFeedbackBar = () => {
```

Comment:
More of something for us to discuss than a suggestion, but I always tend to favor shorter function names inside of respective hooks, classes, etc.  For example, I probably would have name this `show` since we're already inside of the feedback bar hook.

I like the brevity in naming where possible, but I do realize this requires us often to rename the property when destructuring:

```
const { show: showFeedbackBar } = useFeedbackBar();
```

I think both ways are probably fine, but would like to move towards consistent conventions.  We can discuss in Slack and see if there are preferences.  Not something that needs to be updated in this PR.